src/EventStore.Core/Metrics/ElectionsCounterMetric.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Diagnostics.Metrics;
+
+namespace EventStore.Core.Metrics;
+
+public class ElectionsCounterMetric {
+	private Counter<int> ElectionsCounter { get; }
```

Comment:
generally the codebase has private fields rather than private properties, would you mind changing this to a private field